de.dentrassi.pm.aspect.mvnosgi/OSGI-INF/aspect.properties

```diff
@@ -0,0 +1,4 @@
+drone.aspect.id=mvnosgi
+drone.aspect.name=Maven OSGi Channel Aspect
+drone.aspect.version=1.0.0
+drone.aspect.requires=osgi mvn
```

Comment:
I think this one should be `osgi maven.repo`. Since `mvn` is actually the "Maven Extractor". In your functionality would replace it.

I also added HTML help snippet and assigned to the group "maven".